elastic/beats
# filebeat/input/journald: fix field name translation

## What does this PR do?

The field names from journald were not being translated to our format
when sending the event to the output. This commit fixes it.

## Why is it important?

It fixes the Journald event format

## Checklist


~~## Author's Checklist~~

## How to test this PR locally

Run Filebeat with the journald input and check that the event fields are translated as stated on our [documentation](https://www.elastic.co/guide/en/beats/filebeat/8.0/filebeat-input-journald.html#filebeat-input-journald-translated-fields)

From the `filebeat` folder, you can use this yaml as configuration. It points to a journal used on unit tests with a handful of events
```yaml
filebeat.inputs:
- type: journald
  enabled: true
  paths: ["input/journald/testdata/input-multiline-parser.journal"]
  save_remote_hostname: true

output.console:
  codec.json:
    pretty: true
  
processors:
  - add_host_metadata:
      when.not.contains.tags: forwarded
```

## Related issues

- Fixes https://github.com/elastic/beats/issues/30031

~~## Use cases~~
~~## Screenshots~~
~~## Logs~~
